### ISSUE TITLE
No audio feedback for screen readers when changing the slider value.

### ISSUE THREAD

**Issue by obulat** [WordPress/openverse-frontend#1853]
## Problem

No audio feedback for screen readers when changing the slider value.

## Description

Screen readers should get feedback about the current position of the slider when using the left/right arrows, e.g. ‘0 minutes 12 seconds of 3 minutes 43’. The exact wording should be run by a screen reader user for preferences.

## Additional context

Reported by Joe Dolson (https://make.wordpress.org/openverse/2022/09/07/frontend-release-v3-4-8-and-a-call-for-a11y-testing/#comment-95)

## Implementation

- [ ] 🙋 I would be interested in implementing this feature.
